app/utils/AddAssetViewHelper.scala

```diff
@@ -17,136 +17,88 @@
-      },
+      name = asset.orgName,
+      typeLabel = messages(s"entities.asset.shares"),
+      changeUrl = "", //shares.routes.ShareAnswerController.onPageLoad().url,
       removeUrl = ""
     )
   }
 
-  private def parseBusiness(bvm: BusinessAssetViewModel, index: Int): AddRow = {
+  private def renderBusiness(asset: BusinessAssetType, index: Int): AddRow = {
     AddRow(
-      name = bvm.label.getOrElse(defaultName),
-      typeLabel = bvm.`type`.label,
-      changeUrl = if (bvm.status == Completed) {
-        business.routes.BusinessAnswersController.onPageLoad().url
-      } else {
-        business.routes.BusinessNameController.onPageLoad(NormalMode).url
-      },
+      name = asset.orgName,
+      typeLabel = messages(s"entities.asset.business"),
+      changeUrl = "", //business.routes.BusinessNameController.onPageLoad(NormalMode).url,
       removeUrl = ""
     )
   }
 
-  private def parsePartnership(pvm: PartnershipAssetViewModel, index: Int): AddRow = {
+  private def renderPartnership(asset: PartnershipType, index: Int): AddRow = {
     AddRow(
-      name = pvm.label.getOrElse(defaultDescription),
-      typeLabel = pvm.`type`.label,
-      changeUrl = if (pvm.status == Completed) {
-        partnership.routes.PartnershipAnswerController.onPageLoad().url
-      } else {
-        partnership.routes.PartnershipDescriptionController.onPageLoad(NormalMode).url
-      },
+      name = asset.description,
+      typeLabel = messages(s"entities.asset.partnership"),
+      changeUrl = "", //partnership.routes.PartnershipAnswerController.onPageLoad().url,
       removeUrl = ""
     )
   }
 
-  private def parseOther(ovm: OtherAssetViewModel, index: Int): AddRow = {
+  private def renderOther(asset: OtherAssetType, index: Int): AddRow = {
     AddRow(
-      name = ovm.label.getOrElse(defaultDescription),
-      typeLabel = ovm.`type`.label,
-      changeUrl = if (ovm.status == Completed) {
-        other.routes.OtherAssetAnswersController.onPageLoad().url
-      } else {
-        other.routes.OtherAssetDescriptionController.onPageLoad(NormalMode).url
-      },
+      name = asset.description,
+      typeLabel = messages(s"entities.asset.other"),
+      changeUrl = "", //other.routes.OtherAssetAnswersController.onPageLoad().url,
       removeUrl = ""
     )
   }
   
-  private def parseNonEeaBusiness(nebvm: NonEeaBusinessAssetViewModel, index: Int): AddRow = {
+  private def renderNonEEABusiness(asset: NonEeaBusinessType, index: Int): AddRow = {
     AddRow(
-      name = nebvm.label.getOrElse(defaultName),
-      typeLabel = nebvm.`type`.label,
-      changeUrl = if (nebvm.status == Completed) {
-        noneeabusiness.routes.AnswersController.onPageLoad().url
-      } else {
-        noneeabusiness.routes.NameController.onPageLoad(NormalMode).url
-      },
+      name = asset.orgName,
+      typeLabel = messages(s"entities.asset.nonEeaBusiness"),
+      changeUrl = noneeabusiness.routes.NameController.onPageLoad(NormalMode).url,
```

Comment:
Should redirect to answers page

Reply:
Ok, I didn't know that. I'll fix that

Reply:
Actually, Richy will as the Amend is his ticket